CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ For information on changes in released versions of Teku, see the [releases page]
 ## Unreleased Changes
 
 ### Breaking Changes
+- removed `/teku/v1/beacon/states/{state_id}`, as the functionality is in the standard state apis.
```

Comment:
Should be able to remove it from the upcoming breaking changes now (finally).

Reply:
i looked at that and missed it! fair call